web/src/user_group_create.js

```diff
@@ -25,6 +25,46 @@ export function get_name() {
     return created_group_name;
 }
 
+export function toggle_user_group_creation_tabs(next_tab) {
+    if (next_tab) {
+        // Hide user group creation containers and show add subscriber container
+        $("#groups_overlay #choose_member_section").show();
+        $(
+            "#user-group-permission-container, #user-group-description-container,#user-group-name-container",
+        ).hide();
+        $("#groups_overlay .finalize_create_user_group").show();
+        $("#groups_overlay #create_user_group_next_tab").hide();
+        $("#groups_overlay #create_user_group_back_tab").show();
+        $("#groups_overlay .user-group-info-title").text(
+            $t({defaultMessage: "Create user group: add subscribers"}),
+        );
+    } else {
+        // Hide add subscriber container and show user group creation containers
+        $("#groups_overlay #choose_member_section").hide();
+        $(
+            "#user-group-permission-container, #user-group-description-container,#user-group-name-container",
+        ).show();
+        $("#groups_overlay .finalize_create_user_group").hide();
+        $("#groups_overlay #create_user_group_next_tab").show();
+        $("#groups_overlay #create_user_group_back_tab").hide();
+        $("#groups_overlay .user-group-info-title").text(
+            $t({defaultMessage: "Create user group: configure settings"}),
```

Comment:
These changes can be consistent with what we do for streams.

Reply:
to be consistent with what we do in streams. I have imported `user_group_edit.js` in `user_group_create.js` but it is giving me error.

```console
eslint    | /home/sujal/zulip/web/src/user_group_create.js
eslint    |   12:1  error  Dependency cycle detected  import/no-cycle
eslint    |
eslint    | /home/sujal/zulip/web/src/user_group_edit.js
eslint    |   28:1  error  Dependency cycle detected  import/no-cycle

```

Reply:
I think you can move `show_user_group_settings_pane` to `user_group_components` in a prep commit and that should solve this.